### ISSUE TITLE
Support for different languages and locales.

### ISSUE THREAD

**Issue by gendine** [shokanduel/webapp#10]
# Support for different locales and languages

The application should allow the user to change the language and locale of the UI in their settings and save their changes.

Refer to [this](https://angular.io/guide/i18n) for more information.
Angular's [repo](https://github.com/angular/angular/tree/master/packages/common/locales) of common locales.

**Comment by ethanmassie** [comment]
Yup, I have some familiarity with i18n from work.  We will also need to do translation on the backend for any strings that get sent to the frontend.
